UI/card_pile_ui.gd

```diff
@@ -0,0 +1,54 @@
+extends Control
+
+@onready var gridcontainer: GridContainer = $GridContainer
+@onready var parent_node = $".."
+
+#I wanted to use pure_card as card_scene is used in (https://github.com/Paper-2/1M_sub/blob/bc1048d2eddec7aa6205d1bc2be40c519858b40a/Cards/CardContainer.gd#L117-L125)
+@onready var pure_card: PackedScene
```

Comment:
what is this var ?

Reply:
The equivalant to card_scene used in cardcontainer

Reply:
maybe keep the same name so we don't get confused